diff --git a/node_modules/expo-ads-admob/android/build.gradle b/node_modules/expo-ads-admob/android/build.gradle
index d1f33ca..1b697d9 100644
--- a/node_modules/expo-ads-admob/android/build.gradle
+++ b/node_modules/expo-ads-admob/android/build.gradle
@@ -36,10 +36,11 @@ buildscript {
 }
 
 // Creating sources with comments
-task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
-  from android.sourceSets.main.java.srcDirs
-}
+// ─── Gradle 8+ compatible Android Sources JAR ───
+//tasks.register("androidSourcesJar", Jar) {
+//  archiveClassifier.set("androidSources")
+//  from androidSources
+//}
 
 afterEvaluate {
   publishing {
@@ -47,7 +48,7 @@ afterEvaluate {
       release(MavenPublication) {
         from components.release
         // Add additional sourcesJar to artifacts
-        artifact(androidSourcesJar)
+        // artifact(androidSourcesJar)
       }
     }
     repositories {
@@ -59,7 +60,7 @@ afterEvaluate {
 }
 
 android {
-  compileSdkVersion safeExtGet("compileSdkVersion", 31)
+  compileSdkVersion safeExtGet("compileSdkVersion", rootProject.ext.compileSdkVersion)
 
   compileOptions {
     sourceCompatibility JavaVersion.VERSION_11
@@ -71,8 +72,8 @@ android {
   }
 
   defaultConfig {
-    minSdkVersion safeExtGet("minSdkVersion", 21)
-    targetSdkVersion safeExtGet("targetSdkVersion", 31)
+    minSdkVersion safeExtGet("minSdkVersion", rootProject.ext.minSdkVersion)
+    targetSdkVersion safeExtGet("targetSdkVersion", rootProject.ext.targetSdkVersion)
     versionCode 26
     versionName "13.0.0"
   }
